__tests__/app.test.js

```diff
@@ -0,0 +1,62 @@
+const app = require("../mvc/app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/index.js");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("App", () => {
+  describe("GET /api/healthcheck", () => {
+    test("should return 200 status", () => {
```

Comment:
There is no need to test for a 200 explicitly as we check for status code checks in every single test (see line 23). If this line were to fail, the test would fail ands end execution.